homeassistant/components/home_plus_control/__init__.py

```diff
@@ -53,7 +55,7 @@ async def async_setup(hass: HomeAssistant, config: dict) -> bool:
         _LOGGER.debug(
             "No config in configuration.yaml for the Legrand Home+ Control component"
         )
-        return True
+        return False
```

Comment:
We should still return True here. The config flow will handle a missing implementation.